soc/arm/ti_k3/am62x_m4/soc.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Texas Instruments Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <soc.h>
+
+#define ADDR_TRANSLATE_RAT_BASE_ADDR		(0x044200000u)
+#define PINCTRL_BASE_ADDR					(0x4080000u)
```

Comment:
alignment here looks a bit weird?
I think someone else pointed this out as well, not sure.

Reply:
Fixed.